### ISSUE TITLE
Create tests

### ISSUE THREAD

**Issue by mitsukaki** [piranna/logon#4]
This issue is more so I remember to write tests for this module. I ought to make a thread on NodeOS to mark all the modules we have yet to write tests for.

**Comment by piranna** [comment]
> I ought to make a thread on NodeOS to mark all the modules we have yet to write tests for.

Good idea. In the long term we could write a tool to check for tests on all the dependencies, but by the moment a to-do list would be enough :-) Would you want to start the issue there?

**Comment by mitsukaki** [comment]
> Good idea. In the long term we could write a tool to check for tests on all the dependencies, but by the moment a to-do list would be enough :-) Would you want to start the issue there?

I'm not able to at the very moment. If you would be able to that'd be great. Else, when I get home I'll open one

**Comment by piranna** [comment]
Ok... [it was already there](https://github.com/NodeOS/NodeOS/issues/75) :-P
